org.eclipse.scanning.test/src/org/eclipse/scanning/test/scan/nexus/PerScanMonitorTest.java

```diff
@@ -279,7 +277,7 @@ private void checkPerScanMonitors(final ScanModel scanModel, NXinstrument instru
 		}
 	}
 
-	private IRunnableDevice<ScanModel> createStepScan(IScannable<?> monitor,
+	private IRunnableDevice<ScanModel> createStepScan(IScannable<?> perPointMonitor,
 			IScannable<?> perScanMonitor, int... size) throws Exception {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Define and throw a dedicated exception instead of using a generic one. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS00112)